lib/data/multipleBackendGateway.js

```diff
@@ -62,7 +63,28 @@ const multipleBackendGateway = {
         return client.delete(objectGetInfo, reqUids, callback);
     },
 
-    // checkHealth:
+    healthcheck: (log, callback) => {
+        const multBackendResp = {};
+        async.each(Object.keys(clients), (location, cb) => {
+            const client = clients[location];
+            if (client.clientType === 'scality') {
+                client.healthcheck(log, (err, res) => {
+                    if (err) {
+                        multBackendResp[location] = { error: err };
+                    } else {
+                        multBackendResp[location] = { code: res.statusCode,
+                            message: res.statusMessage };
+                    }
+                    cb();
+                });
+            } else {
```

Comment:
please add comment that this is the response for a file or mem backend.  as we add other possible backends, they will have to be addressed.